tests/edx_platform_mock/test_site/urls.py

```diff
@@ -11,6 +11,7 @@
     url(r'^api/$', views.api),
     url(r'^user_api/$', views.api),
     url(r'^notifier_api/$', views.api),
+    url(r'^api/discussion/$', views.api),
```

Comment:
Sorry, Omar, I thought there was a `discussion_api` url prefix.  If not, you don't really need this.  as it's redundent with plain `api/`.

Reply:
![image](https://user-images.githubusercontent.com/645156/41233486-4090e12c-6d92-11e8-9a6f-dce07f4b186a.png)

I'll remove it then.